retriever/lib/engine_tools.py

```diff
@@ -199,7 +199,7 @@ def xml2csv(input_file, outputfile=None, header_values=None, row_tag="row"):
     return outputfile
 
 
-def getmd5(data, data_type='lines', encoding='utf-8'):
+def getmd5(data, data_type='lines', encoding=ENCODING):
```

Comment:
I do not recommend changing this line.

Reply:
It was causing encoding issues when trying to find the `md5` of raw data of `aquatic-animal-excretion`.

Reply:
If you are dealing with this script, you may want to start debugging from 
`retriever install --debug csv  aquatic-animal-excretion`

Reply:
![errr](https://user-images.githubusercontent.com/21103831/58644178-6186ea00-831e-11e9-9f64-3ed6ce3f5f5e.png)
It is giving the same encoding error. Working fine if I change the encoding to `ISO-8859-1` for the script.